dev/doc/critical-bugs

```diff
@@ -0,0 +1,221 @@
+  introduced: 23 May 2006, 9c2d70b, r8845, Herbelin
+  impacted released versions: V8.1-V8.1pl4, V8.2-V8.2pl2, V8.3-V8.3pl2
+  impacted development branches: none
+  impacted coqchk versions: ?
+  fixed in: trunk/master/v8.4 (8082d1faf, 5 Oct 2011, Herbelin), V8.3pl3 (bb582bca2, 5 Oct 2011, Herbelin), v8.2 branch (3333e8d3, 5 Oct 2011, Herbelin), v8.1 branch (a8fc2027, 5 Oct 2011, Herbelin), 
+  found by: Barras
+  exploit: test-suite/failure/inductive4.v
+  GH issue number: none
+  risk: unlikely to be activated by chance
+
+Primitive projections
+
+  component: primitive projections, guard condition
+  summary: check of guardedness of extra arguments of primitive projections missing
+  introduced: 6 May 2014, a4043608f, Sozeau
+  impacted released versions: V8.5-V8.5pl2, 
+  impacted development branches: none
+  impacted coqchk versions: ?
+  fixed in: trunk/master/v8.5 (ba00867d5, 25 Jul 2016, Sozeau)
+  found by: Sozeau, by analyzing bug report #4876
+  exploit: to be done (?)
+  GH issue number: #4876
+  risk: consequence of bug found by chance, unlikely to be exploited by chance (MS?)
+
+  component: primitive projections, guard condition
+  summary: records based on primitive projections became possibly recursive without the guard condition being updated
+  introduced: 10 Sep 2014, 6624459e4, Sozeau (?)
+  impacted released versions: V8.5
+  impacted development branches: none
+  impacted coqchk versions: ?
+  fixed in: trunk/master/v8.5 (120053a50, 4 Mar 2016, Dénès)
+  found by: Dénès exploiting bug #4588
+  exploit: test-suite/bugs/closed/4588.v
+  GH issue number: #4588
+  risk: ?
+
+Conversion machines
+
+  component: "virtual machine" (compilation to bytecode ran by a C-interpreter)
+  summary: collision between constructors when more than 256 constructors in a type
+  introduced: V8.1
+  impacted released versions: V8.1-V8.5pl3, V8.2-V8.2pl2, V8.3-V8.3pl3, V8.4-V8.4pl5
+  impacted development branches: none
+  impacted coqchk versions: none (no virtual machine in coqchk)
+  fixed in: master/trunk/v8.5 (00894adf6/596a4a52, 26-39 Mar 2015, Grégoire), v8.4 (cd2101a39, 1 Apr 2015, Grégoire), v8.3 (a0c7fc05b, 1 Apr 2015, Grégoire), v8.2 (2c6189f61b8, 1 Apr 2015, Grégoire), v8.1 (bb877e5b, 29 Nov 2015, Herbelin)
+  found by: Dénès, Pédrot
+  exploit: test-suite/failure/vm-bug4157.v
+  GH issue number: #4157
+  risk: 
+
+  component: "virtual machine" (compilation to bytecode ran by a C-interpreter)
+  summary: wrong universe constraints
+  introduced: possibly exploitable from V8.1; exploitable at least from V8.5
+  impacted released versions: V8.1-V8.4pl5 unknown, V8.5-V8.5pl3, V8.6-V8.6.1, V8.7.0-V8.7.1
+  impacted development branches: unknown for v8.1-v8.4, none from v8.5
+  impacted coqchk versions: none (no virtual machine in coqchk)
+  fixed in: master (c9f3a6cbe5c41, 12 Feb 2018, PR#6713, Dénès), v8.7 (c058a4182, 15 Feb 2018, Zimmermann), v8.6 (a2cc54c6, 21 Feb 2018, Herbelin), v8.5 (d4d550d0f1a, 21 Feb 2018, Herbelin)
```

Comment:
I'm just the committer for this commit. The listed author is Maxime Dénès.

Reply:
I clarified your role by adding the mention "backport" (here and everywhere else relevant). That'd be ok?

Reply:
Yes